### PR TITLE
Fix for opening gzipped files with different line endings

### PR DESCRIPTION
**Pull request by jkokosar**
* Fix for opening gzipped files with different line endings. Added/modified associated test.
* Introduced utils.py in tools folder
* Introduced gzopen() utils function
* Switched from gzip.open() to gzopen() where appropriate
* Some PEP8 corrections

### CODE REVIEW

**Comment by jkokosar** [comment]
See the discussion regarding the gzopen() here: https://github.com/genialis/resolwe-bio/pull/1

File: resolwe_bio/tools/gen_utils.py [hunk 1]
```diff
@@ -110,3 +110,8 @@ def gen_checkrc(rc, *args):
         ret['proc.error'] = error_msg
 
     return json.dumps(ret)
+
+def gen_warning(value):
```
Comment (mstajdohar) [inline resolwe_bio/tools/gen_utils.py]: Duplicate function

**Comment by jkokosar** [comment]
Fixed!

**Comment by tjanez** [comment]
@mstajdohar, @jkokosar, please include what you've described in the PR also in the commit message next time. E.g., the commit message would look like:
```
Introduced utils.gzopen() to handle different line endings and improve performance.

Fix for opening gzipped files with different line endings.
Added/modified associated test.
Introduced utils.py in tools folder.
Introduced gzopen() utils function.
Switched from gzip.open() to gzopen() where appropriate.
Some PEP8 corrections.
```

**Pull request merged**